mock-up/index.html

```diff
@@ -59,7 +59,7 @@ <h1 class="title-words">Front-End Tools</h1>
     <h1 class="title-words">Back-End Tools</h1>
   </div>
 
-  <div class="techs">
+  <div class="techs-container">
     <div class="tech">
       <img src="assets/rails_icon.png" alt="rails" class="tech-img">
```

Comment:
sorry. I forget to tell you that you must close img tag. `<img />`
This allows us to treat it as if it were well-formed XML. Please fixes all